test/e2e/network_create_test.go

```diff
@@ -348,6 +348,23 @@ var _ = Describe("Podman network create", func() {
 		Expect(nc.OutputToString()).ToNot(ContainSubstring("dnsname"))
 	})
 
+	It("podman network create with internal should not have dnsname", func() {
+		net := "internal-test" + stringid.GenerateNonCryptoID()
+		nc := podmanTest.Podman([]string{"network", "create", "--internal", net})
+		nc.WaitWithDefaultTimeout()
+		defer podmanTest.removeNetwork(net)
+		Expect(nc).Should(Exit(0))
+		// Not performing this check on remote tests because it is a logrus error which does
+		// not come back via stderr on the remote client.
+		//if !IsRemote() {
+		//	Expect(nc.ErrorToString()).To(ContainSubstring("dnsname and internal networks are incompatible"))
+		//}
+		nc = podmanTest.Podman([]string{"network", "inspect", net})
+		nc.WaitWithDefaultTimeout()
+		Expect(nc).Should(Exit(0))
+		Expect(nc.OutputToString()).ToNot(ContainSubstring("dnsname"))
```

Comment:
should contain?

Reply:
@baude with the latest changes from c/common vendored, I see this test fails with this: 

```
Expected
      <string>: [ { "name": "internal-test95c39dc4c847529f78f5805d3ad9a1ef386d2453db8c3e798a4719df3ccf217f", "id": "05359ab2f2869809dce1e266c09d6e90cb33a2d2902b639124b6408974502487", "driver": "bridge", "network_interface": "podman2", "created": "2022-02-07T14:42:05.945465518-05:00", "subnets": [ { "subnet": "10.89.1.0/24", "gateway": "10.89.1.1" } ], "ipv6_enabled": false, "internal": true, "dns_enabled": true, "ipam_options": { "driver": "host-local" } } ]
  to contain substring
      <string>: dnsname
```

Are we still missing something from `podman network inspect` ?

Reply:
the new network stuff uses dns_enabled: bool